test/integration/targets/ios_acl/vars/main.yaml

```diff
@@ -0,0 +1,232 @@
+---
+deleted:
+  commands:
+    - "no ip access-list extended test_acl"
+    - "no ip access-list extended 110"
+    - "no ip access-list extended 123"
+    - "no ipv6 access-list R1_TRAFFIC"
+
+deleted_afi:
+  commands:
+    - "no ip access-list extended 110"
+    - "no ip access-list extended 123"
+    - "no ip access-list extended test_acl"
+
+merged:
+  commands:
+    - "ip access-list standard std_acl"
+    - "deny 192.0.2.0 0.0.0.255"
+    - "ip access-list extended test_acl"
+    - "deny tcp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 eq www fin option traceroute ttl eq 10"
+    - "ip access-list extended 110"
+    - "10 deny icmp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 echo dscp ef ttl eq 10"
+    - "ip access-list extended 123"
+    - "deny tcp 198.51.100.0 0.0.0.255 198.51.101.0 0.0.0.255 eq telnet ack tos 12"
+    - "deny tcp 192.0.3.0 0.0.0.255 192.0.4.0 0.0.0.255 eq www ack dscp ef ttl lt 20"
+    - "ipv6 access-list R1_TRAFFIC"
+    - "deny tcp any eq www any eq telnet ack dscp af11"
+
+replaced:
+  commands:
+    - "no ip access-list extended 110"
+    - "ip access-list extended 110"
+    - "deny tcp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 eq www syn dscp ef ttl eq 10"
+    - "ip access-list extended 150"
+    - "20 deny tcp 198.51.100.0 0.0.0.255 eq telnet 198.51.110.0 0.0.0.255 eq telnet syn dscp ef ttl eq 10"
+
+overridden:
+  commands:
+    - "no ip access-list extended 123"
+    - "no ip access-list extended test_acl"
+    - "no ip access-list extended 110"
```

Comment:
As it appears, we are deleting and then reconfiguring an ACL in the same task. The deletion is redundant, I believe and should be removed.

Reply:
same as above

Reply:
As per the commands here, we remove ACL 110, then again reconfigured it. The first delete command can be avoided.

Reply:
In case of override we delete the already existing acl and then try to configure based on want override config, so am I missing something here